kuka_sunrise_fri_driver/src/hardware_interface.cpp

```diff
@@ -468,7 +468,9 @@ void KukaFRIHardwareInterface::activateFrictionCompensation(double * values) con
 {
   for (int i = 0; i < DOF; i++)
   {
-    values[i] -= (values[i] / fabs(values[i]) * 0.1);
+    if(values[i]!=0.0){
```

Comment:
please add else branch where 0 is also shifted with 0.1 (direction does not count for me)
this whole friction compensation might not make much sense, but before we verify that, edge cases should also be handled